docs/en/administrator-guide/operation/tablet-meta-tool.md

```diff
@@ -93,14 +93,39 @@ Order:
 
 ### Delete header
 
-In order to realize the function of deleting a tablet from a disk of a be.
+In order to realize the function of deleting a tablet from a disk of a BE. Support single delete and batch delete.
 
-Order:
+Single delete:
 
 ```
 ./lib/meta_tool --operation=delete_meta --root_path=/path/to/root_path --tablet_id=xxx --schema_hash=xxx`
 ```
 
+Batch delete:
+
+```
+./lib/meta_tool --operation=batch_delete_meta --tablet_file=/path/to/tablet_file.txt
```

Comment:
will you need to support delete tablet data meanwhile?

Reply:
Maybe at another PR